webpack/redux/actions/Settings/tables.js

```diff
@@ -0,0 +1,85 @@
+import { foremanApi, userId } from '../../../services/api';
```

Comment:
I would argue that this file should live in `webpack/scenes/Settings` as it's similar to `webpack/scenes/Organizations`.  The files in the redux directory will be moved in the future.

Reply:
I went ahead and moved all redux work for tables to the recommended folder you mentioned above.